packages/block-library/src/query/variations.js

```diff
@@ -53,6 +54,9 @@ const variations = [
 			},
 		},
 		scope: [ 'inserter' ],
+		isActive: ( { namespace, query } ) => {
+			return namespace === 'core/posts-list' && query.postType === 'post';
+		},
 	},
 	{
 		name: 'title-date',
```

Comment:
The other variations like Title & Date or Title & Excerpt, would they benefit from `isActive`, too?

Reply:
No, because they use the `scope: [ 'block' ]`, which is used for the placeholder and they essentially are there to add some starting inner blocks.